lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -74,6 +90,22 @@ def build_auth_request(amount, payment, options)
         end.compact
       end
 
+      def build_refund_request(amount, options)
+        { clientReferenceInformation: {}, orderInformation: {} }.tap do |post|
+          add_code(post, options)
+          add_amount(post, amount)
+        end.compact
+      end
+
+      def build_credit_request(amount, payment, options)
+        { clientReferenceInformation: {}, paymentInformation: {}, orderInformation: {} }.tap do |post|
+          add_code(post, options)
+          add_credit_card(post, payment)
+          add_amount(post, amount)
+          add_address(post, payment, options[:billing_address], options, :billTo)
```

Comment:
💬 Comment:

Seems that `add_merchant_description` is missing here ... comparing the actual implementation against the old one

Reply:
Done!